src/ink/tools.py

```diff
@@ -103,7 +103,7 @@ def install_airbyte_repo():
 
     if not os.path.isdir(AIRBYTE_PROJECT_PATH):
         logging.debug(f"Cloning git repo: {AIRBYTE_GIT_REPOSITORY}")
-        cmd = ["git", "clone", "-q"]
+        cmd = ["git", "clone", "-q", "--depth", "1"]
```

Comment:
Might be out of context: I don't know exactly if you are planning to implement other git operation but using this trick with `AIRBYTE_GIT_BRANCH` you don't get the master branch and can't run a `git diff`. I did not find yet a good solution to just checkout two branches: master + a custom feature branch.
Nit: I also made use of `GitPython` instead of using `subprocess` in other project. I also heard that `pygit2` might be better in term of performace.